android/third_party_libs.md

```diff
@@ -10,13 +10,13 @@ Source: <https://github.com/square/retrofit>
 Overview of OkHttp: <http://square.github.io/okhttp> <br/>
 Source for OkHttp: <https://github.com/square/okhttp>
 
-### Otto
+### EventBus
 
-An event bus tailored to Android, Otto supports communication between distinct application components, when the use of a traditional interface is inadequate or cumbersome. 
+An event bus tailored to Android, greenrobot's EventBus supports communication between distinct application components; generally, the event bus should only be used when a traditional interface is inadequate or cumbersome, or an event is broadcast to multiple listeners at once.  
```

Comment:
It would then be a matter of how many activities can be bound to a service. If multiple activities are listening for the service and/or the service is not bound to any particular activity, the event bus would be appropriate. However, if the service should only be active when a certain activity is active, then a connection interface would be better.